docker_templates/templates/snippet/vcs_import.Dockerfile.em

```diff
@@ -1,6 +1,8 @@
 @[if 'vcs' in locals()]@
 @[  if vcs]@
 @[    for i, (imports_name, imports) in enumerate(vcs.items())]@
+if imports['repos'] is None:
+    imports['repos'] = "https://raw.githubusercontent.com/ros2/ros2/$ROS_DISTRO/ros2.repos"
```

Comment:
I like that :+1:. Only thing to keep in mind is that this smippet is now ROS 2 specific (as opposed to project agnostic).
Though we don't expect people to enter this branch much anyway so it shouldn't be a big deal.

Reply:
We can refactor the templates a little more once we settle on the Dockerfile content.